application/common/models/Method.php

```diff
@@ -324,4 +346,39 @@ public static function deleteExpiredUnverifiedMethods()
         }
     }
 
+    /**
+     * Gets all verified methods for user specified by $employeeId
+     * @param string $employeeId
+     * @return String[]
+     */
+    public static function getVerifiedMethods($employeeId)
+    {
+        $method = new Method;
+        return $method->idBrokerClient->listMethod($employeeId);
```

Comment:
If broker returns unverified methods now this'll need updating too

Reply:
Updated.